src/shared/__stories__/drop-target.stories.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import styled from "react-emotion";
+import { storiesOf } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+
+import DropTarget from "../components/drop-target";
+import { ContainedButton } from "../components/buttons";
+
+const DropTargetContent = styled.div`
+  align-content: center;
+  align-items: center;
+  display: grid;
+  grid-gap: 20px;
+  justify-content: center;
+  justify-items: center;
+`;
+
+export const actions = {
+  onDrop: action("onDrop")
+};
+
+const stories = storiesOf("DropTarget", module);
+
+stories.add("default", () => (
+  <DropTarget {...actions}>
+    <DropTargetContent>
+      <ContainedButton>Add files</ContainedButton>
+      <span>or drag and drop files here</span>
+    </DropTargetContent>
+  </DropTarget>
+));
+
+stories.add("customization", () => (
```

Comment:
I mention this elsewhere, but it would be great to add more stories showing this component's flexibility. The hope is we will be able to use a generic drag and drop for (1) in-body elements where we need to give some clear direction that you can just drag something in, (2) elements where it's not 100% obvious but a user might expect _somethign_ like that to happen (Think a files drawer in the notebook, for example), and (3) a full-body drag and drop. I don't think (3) is probably achievable with what you have right now (and wasn't in your original figma sketches at any rate), but it does seem like (1) and (2) are.